starter-code/public/scripts/article.js

```diff
@@ -85,11 +84,16 @@
 
   Article.numWordsByAuthor = () => {
     return Article.allAuthors().map(author => {
-      // TODO: Transform each author string into an object with properties for
+      // DONE: Transform each author string into an object with properties for
       // the author's name, as well as the total number of words across all articles
       // written by the specified author.
-
-    })
+      var authorStats = {
```

Comment:
don't need to create new object variable, you can just return an object. for instance:

return {
   author: ...,
   stats: ....
}